Source/quests.cpp

```diff
@@ -287,31 +273,34 @@ void InitQuests()
 
 	int q = 0;
 	for (auto &quest : Quests) {
-		if (!gbIsMultiplayer || !QuestData[q].isSinglePlayerOnly) {
-			quest._qtype = QuestData[q]._qdtype;
-			if (gbIsMultiplayer) {
-				quest._qlevel = QuestData[q]._qdmultlvl;
-				if (!delta_quest_inited(initiatedQuests)) {
-					quest._qactive = QUEST_INIT;
-					quest._qvar1 = 0;
-					quest._qlog = false;
-				}
-				initiatedQuests++;
-			} else {
+		quest._qidx = static_cast<quest_id>(q);
+		auto &questData = QuestData[q];
+		q++;
+
+		quest._qactive = QUEST_NOTAVAIL;

```

Comment:
Probably worth bringing up the constant assignments for _qcar1, _qvar2, _qlog, position. They could potentially be made into a constructor later and it'll be easier to identify them if they're together.